localscope/__init__.py

```diff
@@ -296,8 +304,12 @@ def _safely_get_closure_vars(func):  # pragma: no cover
             try:
                 nonlocal_vars[var] = cell.cell_contents
             except ValueError as ex:
-                if not (var == "__class__" and str(ex) == "Cell is empty"):
-                    raise
+                if str(ex) == "Cell is empty":
+                    nonlocal_vars[var] = EmptyCell
+                else:
+                    raise LocalscopeException(
```

Comment:
Isn't the else case happening when the cell is not empty - but still raised a ValueError for some reason? Then I think the message in the LocalscopeException is confusing.